process/transactionProcessor.go

```diff
@@ -30,16 +30,17 @@ const TransactionSimulatePath = "/transaction/simulate"
 // MultipleTransactionsPath defines the multiple transactions send path of the node
 const MultipleTransactionsPath = "/transaction/send-multiple"
 
-// UnknownStatusTx defines the response that should be received from an observer when transaction status is unknown
-const UnknownStatusTx = "unknown"
-
 const (
-	withResultsParam    = "?withResults=true"
-	checkSignatureFalse = "?checkSignature=false"
-	bySenderParam       = "&by-sender="
-	fieldsParam         = "?fields="
-	lastNonceParam      = "?last-nonce=true"
-	nonceGapsParam      = "?nonce-gaps=true"
+	withResultsParam       = "?withResults=true"
+	checkSignatureFalse    = "?checkSignature=false"
+	bySenderParam          = "&by-sender="
+	fieldsParam            = "?fields="
+	lastNonceParam         = "?last-nonce=true"
+	nonceGapsParam         = "?nonce-gaps=true"
+	txCompleted            = "completedTxEvent"
+	txFailed               = "signalError"
+	scDeploy               = "SCDeploy"
```

Comment:
The constants can be prefixed or suffixed with "event" (or something similar). Furthermore, `moveBalanceTransaction` can be renamed to include the prefix or suffix "processing type".